packages/react-examples/.storybook/preview-loader.js

```diff
@@ -2,12 +2,11 @@ import * as fs from 'fs-extra';
 import * as path from 'path';
 
 /**
- * Replace the PACKAGE_NAME token in preview.js with the actual current package name
- * (or error if the package doesn't have examples).
+ * Webpack loader that replaces the PACKAGE_NAME token in preview.js with the actual current
+ * package name (or errors if the package doesn't have examples).
  *
- * Also if this is the storybook for `@fluentui/react`, replace the REACT_DEPS token with the actual
- * list of deps which have their own examples but should be included in the suite package storybook.
- * @this {{addDependency: (file: string)=>void;callback: ((err: Error | null)=>void)}} loader context - https://webpack.js.org/api/loaders/#the-loader-context
+ * @see https://webpack.js.org/api/loaders/
+ * @this {import('webpack').LoaderContext<{}>}
```

Comment:
This might have been one of the types that went missing in webpack 5 and have now been added back